libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotProactiveMessageHandler.cs

```diff
@@ -36,7 +36,7 @@ public class BotProactiveMessageHandler : BotMessageHandlerBase
                 conversationReference = BotMessageHandlerBase.BotMessageSerializer.Deserialize<ConversationReference>(bodyReader);
             }
 
-            await botFrameworkAdapter.ContinueConversationAsync(botAppId, conversationReference, botCallbackHandler, cancellationToken);
+            await botFrameworkAdapter.ContinueConversationAsync(botAppId, conversationReference, botCallbackHandler, cancellationToken).ConfigureAwait(false);
```

Comment:
See earlier comment on not using `.ConfigureAwait(false)` in ASP.NET specific components.